addons/event/__manifest__.py

```diff
@@ -23,7 +23,10 @@
         'security/ir.model.access.csv',
         'views/event_menu_views.xml',
         'views/event_ticket_views.xml',
-        'views/event_views.xml',
+        'views/event_mail_views.xml',
```

Comment:
First commit should be a [MOV] (ref is more when we refactor and change code / feature). You can also explain why we do it: to have one file per model, easing finding of views and following future changes, shorter files, ...